types/yaireo__tagify/index.d.ts

```diff
@@ -1130,6 +1149,12 @@ declare class Tagify<T extends Tagify.BaseTagData = Tagify.TagData> {
      */
     removeAllTags(opts?: Tagify.RemoveAllTagsOptions): void;
 
+    /**
+     * Get `value` (array of tag data) without properties that are only used internally.
+     * @return A list of tag data without `__isValid`, `__removed`, `__originalData`, and `__originalHTML` properties.
+     */
+    getCleanValue(): T[];
```

Comment:
Since these properties are internal, I would not even mention them in the docs (they might change anytime) and just mention `A list of tag data without internal properties (that usually start with two underscores)`